app/controllers/api/v2/environments_controller.rb

```diff
@@ -61,6 +61,13 @@ def destroy
       def allowed_nested_id
         %w(puppetclass_id location_id organization_id)
       end
+
+      def find_resource
```

Comment:
Is it preferable to reconfigure friendly_id's reserved words list?  Since the extra parameterise concern overrides the parameter to be id-name anyway, reserving "new", "edit" etc. doesn't make a lot of sense to me, since we won't ever hit them.

Removing friendly_id from one model, when the same problem is going to apply to almost every other friendly_id enabled model, then reimplementing find_resource seems like a slippery slope to creating copy/paste and non-DRY code.